config/checker/java.util.astub

```diff
@@ -8,6 +8,10 @@ import org.checkerframework.dataflow.qual.Pure;
 
 package java.util;
 
+class Objects {
+  static <T> @NonNull T requireNonNull(@Nullable T obj);
```

Comment:
Required to make sure that the code generated for interpolations in`-AcallToStringExplicitlyInInterpolations` mode (which uses `requireNonNull` under the hood) doesn't raise warnings with Checker